lua/astrocommunity/motion/marks-nvim/init.lua

```diff
@@ -0,0 +1,15 @@
+return {
+  "chentoast/marks.nvim",
+  event = "BufEnter",
```

Comment:
Considering the different excluded filetypes, wouldn't `event = "User AstroFile"` make more sense?

Reply:
ok